### PR TITLE
Feature file info dialog

### PR DESCRIPTION
**Pull request by subkanthi**

### CODE REVIEW

**Comment by gsantner** [comment]
something seems to be broken in this PR, did you add some wrong branch? (6 month old commits etc)

**Comment by subkanthi** [comment]
Not sure why it pulled all the commits, I will create another branch will copy over the commits.

**Comment by gsantner** [comment]
:D You can also just backup files, reset repo and copy back, if something broke

File: app/src/main/java/net/gsantner/markor/ui/FileInfoDialog.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+        TextView textView = root.findViewById(R.id.ui__filesystem_item__description);
+        textView.setText(file.getName());
+
+        TextView locationView = root.findViewById(R.id.ui__filesystem_item__location_description);
+        locationView.setText(file.getAbsolutePath());
+
+        TextView sizeView = root.findViewById(R.id.ui__filesystem_item__size_description);
+        long totalSizeBytes = file.getTotalSpace();
+        String humanReadableSize = android.text.format.Formatter.formatShortFileSize(activityContext, totalSizeBytes);
```
Comment (gsantner) [inline app/src/main/java/net/gsantner/markor/ui/FileInfoDialog.java]: you should be able to get context from  root.getContext(); no need to store activityContext ref I think

File: app/src/main/java/net/gsantner/markor/activity/FilesystemListFragment.java [hunk 2]
```diff
@@ -562,11 +563,24 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
...
                 default:
                     return false;
             }
         }
 
+        /**
+         * Show file information dialog.
+         * @param file
+         */
+        private void showFileInfoDialog(File file) {
```
Comment (gsantner) [inline app/src/main/java/net/gsantner/markor/activity/FilesystemListFragment.java]: you can put those 2 lines in the switch-case

File: app/src/main/java/net/gsantner/markor/ui/FileInfoDialog.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+import android.support.v7.app.AlertDialog;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import net.gsantner.markor.R;
+import net.gsantner.markor.util.AppCast;
```
Comment (gsantner) [inline app/src/main/java/net/gsantner/markor/ui/FileInfoDialog.java]: Dont forget to use auto-reformat. I'm sure you don't have in this case

**Comment by gsantner** [comment]
See comments for first review iteration :)

closing this for now, as the source branch seems to be broken

**Pull request closed without merging**